.hadolint.yaml

```diff
@@ -0,0 +1,3 @@
+ignored:
```

Comment:
better to ignore in the docker file with a comment next to it if we can?

Reply:
Implemented exclusions inline.